SMLHelper/SMLHelper.csproj

```diff
@@ -67,6 +67,10 @@
   </ItemGroup>
   <ItemGroup>
     <Compile Include="Crafting\NodeFamily.cs" />
+    <Compile Include="FishFramework\CustomFish.cs" />
```

Comment:
At the moment, there's only one file in the `FishFramework` folder. Could that be changed, or is it fine that way?